src/components/Offramp/Success.view.tsx

```
@@ -32,6 +32,22 @@ export const OfframpSuccessView = ({
         blockExplorerUrl = utils.getExplorerUrl(claimLinkData.chainId)
     }
 
+    const calculateAmount = () => {
+        if (offrampType === _consts.OfframpType.CASHOUT) {
+            // add safeguards for usd value
+            if (!usdValue || isNaN(parseFloat(usdValue))) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)